src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -1228,6 +1238,7 @@ public static void initClientHooks(Minecraft mc, ReloadableResourceManager resou
         initializedClientHooks = true;
 
         ForgeGameTestHooks.registerGametests();
+        MinecraftForge.EVENT_BUS.post(new RegisterSpriteSourceTypesEvent(SPRITE_SOURCE_TYPES_MAP));
```

Comment:
This is the wrong EventBus.
As you can see on all lines below we currently are still using the mod bus for this, and for now that should stay the case.